tests/unit/schema/test.firefox-schemas-import.js

```diff
@@ -23,6 +23,7 @@ import {
 
 // Get a reference to unlinkSync so it won't get stubbed later
 const { unlinkSync } = fs;
+const FAKE_ZIP_FILE_PATH = './tmp/mozilla-central.zip'
```

Comment:
@marchiore instead of using a tmp directory relative to the repository (which may not even exist when the tests are being executed), we could use ["tmp-promise"](https://www.npmjs.com/package/tmp-promise) to get a temporary directory.

We do already have "tmp-promise" in the dev dependencies, because it is being used in [one of the scripts defined in this repository](https://github.com/mozilla/addons-linter/blob/d9ddd34615a3a2d8d50386c0b000eff653064fae/tests/integration/run-as-production-env.js#L8)), and so it is not even a new dependency.

The change suggested should also fix the failure triggered by this version in the CI job: https://travis-ci.org/github/mozilla/addons-linter/jobs/668838092#L233